ios/src/TGMapViewController.mm

```diff
@@ -205,6 +205,32 @@ - (void)pickFeatureAt:(CGPoint)screenPosition
     });
 }
 
+- (void)pickMarkerAt:(CGPoint)screenPosition
+{
+    if (!self.map && !self.mapViewDelegate) { return; }
+
+    screenPosition.x *= self.contentScaleFactor;
+    screenPosition.y *= self.contentScaleFactor;
+
+    self.map->pickMarkerAt(screenPosition.x, screenPosition.y, [screenPosition, self](const Tangram::MarkerPickResult* markerPickResult) {
+        CGPoint position = CGPointMake(0.0, 0.0);
+
+        if (!markerPickResult) {
+            if ([self.mapViewDelegate respondsToSelector:@selector(mapView:didSelectMarker:atScreenPosition:)]) {
+                [self.mapViewDelegate mapView:self didSelectMarker:0 atScreenPosition:position];
```

Comment:
After thinking about this for a bit, what are your thoughts on having a "success" and "error" callback here, rather than reusing the same method and passing along a zero? We could still pass along the marker id of zero if we want, but I feel like it would be more obvious to the implementing developers what actually happened.